azkaban-common/src/main/java/azkaban/jobtype/JobTypePluginSet.java

```diff
@@ -128,4 +137,11 @@ public void addPluginJobProps(final String jobTypeName, final Props props) {
   public void addPluginLoadProps(final String jobTypeName, final Props props) {
     this.pluginLoadPropsMap.put(jobTypeName, props);
   }
+
+  /**
+   * Adds plugins private properties used by the plugin
+   */
+  public void addPLuginPrivateProps(final String jobTypeName, final Props props) {
```

Comment:
nit: Can you make the "L" in addPLuginPrivateProps lowercase?

Reply:
done